packages/lesswrong/themes/stylePiping.ts

```diff
@@ -244,6 +244,72 @@ const footnoteStyles = (theme: ThemeType): JssStyles => ({
   },
 });
 
+const collapsibleSectionStyles = (theme: ThemeType): JssStyles => ({
+  '& .detailsBlock': {
+    // This conflicts with a CkEditor style on `.ck .ck-editor__nested-editable`
+    // that tries to turn border off and on to indicate selection. Use
+    // !important to ensure it's visible.
+    border: theme.palette.border.normal+' !important',
+    borderRadius: 8,
+    marginTop: 8,
+    marginBottom: 8,
+  },
+  '& .detailsBlockTitle': {
```

Comment:
I wonder if this should have `user-select: none`? It's a little annoying that opening/closing the section can select the title text, but maybe it's worth it for the sake of being able to select/copy/paste the text?